Basic-Car-Maintenance/Shared/Dashboard/Views/DashboardView.swift

```diff
@@ -106,14 +75,16 @@ struct DashboardView: View {
                     }
                 }
             }
-            .task {
-                await viewModel.getMaintenanceEvents()
-                await viewModel.getVehicles()
-            }
             .sheet(isPresented: $isShowingAddView) {
                 makeAddMaintenanceView()
             }
         }
+        .onAppear {
+            Task {
+                await viewModel.getMaintenanceEvents()
+                await viewModel.getVehicles()
+            }
+        }
```

Comment:
Why is this changed from being in `.task` to being in `.onAppear`?

Reply:
What issues?

Reply:
I didn't have real issues, so we should be able to keep `.task`